docs/src/joint/api/connectors/smooth.html

```diff
@@ -1,8 +1,32 @@
-<p>The <code>'smooth'</code> connector interpolates route points using a cubic bezier curve.</p>
+<p>The <code>'smooth'</code> connector interpolates route points using a cubic bezier curve. Without a route it uses options to render the link.</p>
```

Comment:
Since options can be considered if a route is provided too, you could say something like _"...curve. Additional options can be provided to render the link"_ or _"...curve. Additional options can be provided to compliment link rendering."_

Reply:
In the current implementation options don't matter when route points exist. I inherited this from previous implementation.

Reply:
Okay. I think it is fine so.